concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
It's doesn't run in parallel with tests. post-deploy has `passed: ['cf-deploy']` dependency and `avail-tests` are stopped when `cf-deploy` has finished.